roles/ciao-compute/README.md

```diff
@@ -1,33 +1,36 @@
-ciao-compute
-=========
+# ciao-compute
 Ansible role to install the compute node for a CIAO cluster
 
-Requirements
-------------
-None
+## Requirements
+- A previously setup of a CIAO controller node.
```

Comment:
This message looks different format than the ones you present later (e.g: using different words 'server', 'node', 'service) this may lead to confusion so IMHO would be better to stick to a single word here and in other `Requirements` sections